recipes/libproperties/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
I can not understand, why we need put the test_package/CMakeLists.txt in conan-center-index? How about put then in my library?

Reply:
The test_package is here to test whether the built library actually works and links.

Reply:
Thanks. How to let the conan CI system to find the test package from my library? Are there some repos as the example?

Reply:
The test sources should be committed to the conan-center-index repo.
They can be very small, and without error checking.
You can look at literally every package from this repo: `recipes/*/all/test_package`.

I clicked on this one at random: https://github.com/conan-io/conan-center-index/tree/master/recipes/podofo/all/test_package

Reply:
Is the code under test_packge need to run the unit tests?

Reply:
No, it just needs to test whether we can include the headers + link.
It is expected that the library is working..

Reply:
@madebr How the conan client pass the options `shared` to the CMakeLists.txt of the target package?

Reply:
options fPIC and shared are handled automatically by the cmake build helper
See https://docs.conan.io/en/latest/reference/conanfile/attributes.html#options

fPIC is passed using the cmake variable [`CMAKE_POSITION_INDEPENDENT_CODE](https://cmake.org/cmake/help/latest/variable/CMAKE_POSITION_INDEPENDENT_CODE.html)
shared using [`BUILD_SHARED_LIBS`](https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html`)